### DISCUSSION TITLE
Upgrade 4.5.1 breakes existing entities?

### DISCUSSION THREAD

**Discussion by w-wttsl** [joleys/niko-home-control-II#198] [Q&A] [answered]
Since upgrade to 4.5.1 all existing NHC II related entities show: "This entity is no longer being provided by the nhc2 integration"

Connectivity to NHC controller is fine. Debug logging shows activity on the controller when lights are turned on or off. Ex: 2024-12-09 12:44:57.291 DEBUG (Thread-3 (_thread_main)) [custom_components.nhc2.nhccoco.devices.device] XXXXXXXXX changed

This activity is not represented on the device in HA. All logs shown on the device config page in 'Logbook' section date back prior to the 4.5.1 upgrade

I am not sure how to troubleshoot this issue. Any ideas? Anyone having similar issues?

**Comment by tijsverkoyen** [comment]
Which version of Home Assistant are you using?

**Reply by w-wttsl** [reply]
That is

Core: 2024.8.3
Frontend: 20240809.0

(in docker if relevant)

**Comment by tijsverkoyen** [comment, marked as answer]
You should update. 
The current version of Home Assistant is 2024.12

**Reply by w-wttsl** [reply]
Confirmed!

My bad.
